samples/ImageScanning/README.md

```diff
@@ -0,0 +1,249 @@
+# Image Scanning Windows Vision Skills samples
+
+These samples will show you how to use the set of skills contained in the Image Scanning Windows Vision Skill NuGet package to create apps that can achieve productivity scenarios related to scanning content. 
+- [C# UWP sample app](./cs/ImageScanningSample_UWP)
+- [Win32 C++/Winrt Desktop console app](./cpp/ImageScanningSample_Desktop)
+- [.Net Core 3.0 C# console app](./cs/ImageScanningSample_NetCore3)
+
+The skills bundled in this package are:
```

Comment:
Consider:
```markdown
## Included Skills

### CurvedEdgesDetector

Seeks within an image the pixels that constitute the curved edges composing the contour of a given quad and returns their coordinates.

| Input image and base quad specified | Ouput detected curved edge |
| ----------------------------------- | -------------------------- |
| ![Screenshot of CurvedEdgesDetector inputs](./doc/CurvedEdgesDetector2.jpg) | ![Screenshot of CurvedEdgesDetector outputs](./doc/CurvedEdgesDetector3.jpg) |

### ImageCleaner
...

```
# Result:
## Included Skills

### CurvedEdgesDetector

Seeks within an image the pixels that constitute the curved edges composing the contour of a given quad and returns their coordinates.

| Input image and base quad specified | Ouput detected curved edge |
| ----------------------------------- | -------------------------- |
| ![Screenshot of CurvedEdgesDetector inputs](./doc/CurvedEdgesDetector2.jpg) | ![Screenshot of CurvedEdgesDetector outputs](./doc/CurvedEdgesDetector3.jpg) |

### ImageCleaner

...

Reply:
cool yeah I can use that. I'll keep the anchor with a different name though as there is another ### CurvedEdgesDetector later on